app/scripts/lib/createMetaRPCHandler.js

```diff
@@ -20,6 +52,18 @@ const createMetaRPCHandler = (api, outStream) => {
     let error;
     try {
       result = await api[data.method](...data.params);
+
+      if (
+        store &&
+        // TODO: add any methods we know to not change state
+        !['getState'].includes(data.method)
+      ) {
+        const versionData = await localStore.get();
+        await persistData(
+          { data: store.getState(), meta: versionData?.meta },
+          localStore,
+        );
```

Comment:
What is the intent of using meta from versionData (localStore) versus store? I don't see it initiated anywhere so when would it get set?

Reply:
I guess the answer here is the same as my eureka moment with the whole overwriting ... localStore.get *gets* the store from local store where it has always persisted and will have the current meta / version info from the extension.

Reply:
This comment can be resolved by just adding a little explainer comment